packages/salesforcedx-vscode-apex-replay-debugger/src/utils/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as path from 'path';
+import { workspace, WorkspaceFolder } from 'vscode';
+
+export function hasRootWorkspace(ws: typeof workspace = workspace) {
+  return ws && ws.workspaceFolders && ws.workspaceFolders.length > 0;
+}
+
+export function getRootWorkspace(): WorkspaceFolder {
+  return hasRootWorkspace()
+    ? workspace.workspaceFolders![0]
+    : ({} as WorkspaceFolder);
+}
+
+export function getRootWorkspacePath(): string {
+  return getRootWorkspace().uri ? getRootWorkspace().uri.fsPath : '';
+}
+
+export function getLogDirPath(): string {
```

Comment:
Everything except for `getLogDirPath` is available on the new https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-utils-vscode/src/workspaces/rootWorkspace.ts . Let use that one instead of this, if it makes sense to move `getLogDirPath` to the `salesforcedx-utils-vscode` module let's do that too.